go/store/datas/commit.go

```diff
@@ -32,17 +32,19 @@ import (
 )
 
 const (
-	ParentsField = "parents"
-	ValueField   = "value"
-	MetaField    = "meta"
-	commitName   = "Commit"
+	ParentsField     = "parents"
+	ParentsListField = "parentsList"
```

Comment:
cant remember.  should noms field names be snake case?  Maybe there really isn't much precedence.

Reply:
I couldn't find much precedence either way. Happy to change it...`"parents_list"` looks a bit better imo.

Reply:
Went ahead and changed it.